query/src/datasources/system/configs_table_test.rs

```diff
@@ -0,0 +1,88 @@
+// Copyright 2021 Datafuse Labs.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use common_exception::Result;
+use common_planners::*;
+use common_runtime::tokio;
+use futures::TryStreamExt;
+use pretty_assertions::assert_eq;
+
+use crate::catalogs::Table;
+use crate::clusters::Cluster;
+use crate::configs::Config;
+use crate::datasources::system::configs_table::ConfigsTable;
+use crate::sessions::SessionManager;
+
+#[tokio::test(flavor = "multi_thread", worker_threads = 1)]
+async fn test_configs_table() -> Result<()> {
+    let config = Config::default();
+    let cluster = Cluster::empty();
+
+    let sessions = SessionManager::from_conf(config, cluster)?;
+    let test_session = sessions.create_session("TestSession")?;
+    let ctx = test_session.create_context();
+    ctx.get_settings().set_max_threads(8)?;
+
+    let table = ConfigsTable::create();
+    let source_plan = table.read_plan(
+        ctx.clone(),
+        &ScanPlan::empty(),
+        ctx.get_settings().get_max_threads()? as usize,
+    )?;
+
+    let stream = table.read(ctx, &source_plan).await?;
+    let result = stream.try_collect::<Vec<_>>().await?;
+    let block = &result[0];
+    assert_eq!(block.num_columns(), 4);
+    assert_eq!(block.num_rows(), 30);
+
+    let expected = vec![
```

Comment:
What if we add one config, the tests will be failed. So I suggest just to test some common values?

Reply:
any method can be invoked to get rows on result?

Reply:
Like some column names :)